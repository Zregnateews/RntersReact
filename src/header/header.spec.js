import React from 'react';
import { render } from '@testing-library/react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Header } from './header';

configure({adapter: new Adapter()});

describe('Header', () => {
  it('renders correctly', () => {
    const wrapper = render(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
})
