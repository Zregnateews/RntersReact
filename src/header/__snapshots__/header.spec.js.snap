// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <nav
          class="navbar navbar-fixed-top "
        >
          <div
            class="container-fluid"
          >
            <div
              class="navbar-header"
            >
              <div
                class="navbar-brand"
              >
                <a
                  href="/pt"
                >
                  <img
                    alt="Logo simple small"
                    src="https://d3urocn5qjrv7i.cloudfront.net/assets/logo_simple_small-45464c79c00c4d2f0a0e8e10c6b647eb2b8b569ed4abda9a906fca3f72d6e347.png"
                  />
                </a>
                 
              </div>
              <button
                aria-expanded="false"
                class="navbar-toggle collapsed"
                data-target="#navbar-collapse-menu"
                data-toggle="collapse"
                type="button"
              >
                <i
                  class="fas fa-ellipsis-h"
                />
              </button>
            </div>
            <div
              class="collapse navbar-collapse"
              id="navbar-collapse-menu"
            >
              <ul
                class="nav navbar-nav navbar-right"
              >
                <li
                  class="btn_navbar navbar_option"
                  id="btn_become_owner"
                >
                  <a
                    class="monetize_btn"
                    href="/pt/pages/make_money"
                  >
                    Adicionar artigos
                  </a>
                </li>
                <li
                  class="btn_navbar not_visible_mobile_lg"
                >
                  <div
                    class="dropdown"
                  >
                    <a
                      aria-expanded="false"
                      aria-haspopup="true"
                      class="dropdown-toggle"
                      data-toggle="dropdown"
                      role="button"
                    >
                      <span>
                        Saber mais
                      </span>
                       
                      <span
                        class="caret"
                      />
                    </a>
                    <ul
                      class="dropdown-menu"
                      id="know_more_selector"
                    >
                      <li>
                        <a
                          class="navbar_option"
                          href="/pt/pages/how_it_works"
                          id="how_it_works"
                        >
                          Como funciona
                        </a>
                      </li>
                      <li>
                        <a
                          class="navbar_option"
                          href="/pt/pages/make_money"
                          id="make_money"
                        >
                          Owner: Lucrar com o que está parado
                        </a>
                      </li>
                      <li>
                        <a
                          class="navbar_option"
                          href="/pt/pages/rent_stuff"
                          id="rent_stuff"
                        >
                          Renter: Como alugar os meus artigos
                        </a>
                      </li>
                      <li>
                        <a
                          class="navbar_option"
                          href="/pt/pages/security"
                          id="security"
                        >
                          Segurança
                        </a>
                      </li>
                    </ul>
                  </div>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg navbar_option"
                  id="btn_how_it_works"
                >
                  <a
                    href="/pt/pages/how_it_works"
                  >
                    Como funciona
                  </a>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg navbar_option"
                  id="btn_make_money"
                >
                  <a
                    href="/pt/pages/make_money"
                  >
                    Owner: Lucrar com o que está parado
                  </a>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg navbar_option"
                  id="btn_rent_stuff"
                >
                  <a
                    href="/pt/pages/rent_stuff"
                  >
                    Renter: Como alugar os meus artigos
                  </a>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg navbar_option"
                  id="btn_security"
                >
                  <a
                    href="/pt/pages/security"
                  >
                    Segurança
                  </a>
                </li>
                <li
                  class="btn_navbar"
                >
                  <a
                    class="navbar_option"
                    href="/pt/users/sign_in"
                    id="btn_sign_in"
                  >
                    Entrar
                  </a>
                </li>
                <li
                  class="btn_navbar"
                >
                  <a
                    class="navbar_option"
                    href="/pt/users/sign_up"
                    id="btn_sing_up"
                  >
                    Registar
                  </a>
                </li>
                <li
                  class="btn_navbar not_visible_mobile_lg"
                >
                  <div
                    class="dropdown"
                  >
                    <a
                      aria-expanded="false"
                      aria-haspopup="true"
                      class="dropdown-toggle"
                      data-toggle="dropdown"
                      role="button"
                    >
                      <span>
                        pt
                      </span>
                       
                      <span
                        class="caret"
                      />
                    </a>
                    <ul
                      class="dropdown-menu"
                      id="language_selector"
                    >
                      <li>
                        <a
                          data-turbolinks="false"
                          href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                        >
                          Português
                        </a>
                      </li>
                      <li>
                        <a
                          data-turbolinks="false"
                          href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                        >
                          English
                        </a>
                      </li>
                    </ul>
                  </div>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg"
                >
                  <a
                    data-turbolinks="false"
                    href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                  >
                    Português
                  </a>
                </li>
                <li
                  class="btn_navbar only_visible_mobile_lg"
                >
                  <a
                    data-turbolinks="false"
                    href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                  >
                    English
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </nav>
        <div
          id="flash_messages-anchor"
        />
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <nav
        class="navbar navbar-fixed-top "
      >
        <div
          class="container-fluid"
        >
          <div
            class="navbar-header"
          >
            <div
              class="navbar-brand"
            >
              <a
                href="/pt"
              >
                <img
                  alt="Logo simple small"
                  src="https://d3urocn5qjrv7i.cloudfront.net/assets/logo_simple_small-45464c79c00c4d2f0a0e8e10c6b647eb2b8b569ed4abda9a906fca3f72d6e347.png"
                />
              </a>
               
            </div>
            <button
              aria-expanded="false"
              class="navbar-toggle collapsed"
              data-target="#navbar-collapse-menu"
              data-toggle="collapse"
              type="button"
            >
              <i
                class="fas fa-ellipsis-h"
              />
            </button>
          </div>
          <div
            class="collapse navbar-collapse"
            id="navbar-collapse-menu"
          >
            <ul
              class="nav navbar-nav navbar-right"
            >
              <li
                class="btn_navbar navbar_option"
                id="btn_become_owner"
              >
                <a
                  class="monetize_btn"
                  href="/pt/pages/make_money"
                >
                  Adicionar artigos
                </a>
              </li>
              <li
                class="btn_navbar not_visible_mobile_lg"
              >
                <div
                  class="dropdown"
                >
                  <a
                    aria-expanded="false"
                    aria-haspopup="true"
                    class="dropdown-toggle"
                    data-toggle="dropdown"
                    role="button"
                  >
                    <span>
                      Saber mais
                    </span>
                     
                    <span
                      class="caret"
                    />
                  </a>
                  <ul
                    class="dropdown-menu"
                    id="know_more_selector"
                  >
                    <li>
                      <a
                        class="navbar_option"
                        href="/pt/pages/how_it_works"
                        id="how_it_works"
                      >
                        Como funciona
                      </a>
                    </li>
                    <li>
                      <a
                        class="navbar_option"
                        href="/pt/pages/make_money"
                        id="make_money"
                      >
                        Owner: Lucrar com o que está parado
                      </a>
                    </li>
                    <li>
                      <a
                        class="navbar_option"
                        href="/pt/pages/rent_stuff"
                        id="rent_stuff"
                      >
                        Renter: Como alugar os meus artigos
                      </a>
                    </li>
                    <li>
                      <a
                        class="navbar_option"
                        href="/pt/pages/security"
                        id="security"
                      >
                        Segurança
                      </a>
                    </li>
                  </ul>
                </div>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg navbar_option"
                id="btn_how_it_works"
              >
                <a
                  href="/pt/pages/how_it_works"
                >
                  Como funciona
                </a>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg navbar_option"
                id="btn_make_money"
              >
                <a
                  href="/pt/pages/make_money"
                >
                  Owner: Lucrar com o que está parado
                </a>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg navbar_option"
                id="btn_rent_stuff"
              >
                <a
                  href="/pt/pages/rent_stuff"
                >
                  Renter: Como alugar os meus artigos
                </a>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg navbar_option"
                id="btn_security"
              >
                <a
                  href="/pt/pages/security"
                >
                  Segurança
                </a>
              </li>
              <li
                class="btn_navbar"
              >
                <a
                  class="navbar_option"
                  href="/pt/users/sign_in"
                  id="btn_sign_in"
                >
                  Entrar
                </a>
              </li>
              <li
                class="btn_navbar"
              >
                <a
                  class="navbar_option"
                  href="/pt/users/sign_up"
                  id="btn_sing_up"
                >
                  Registar
                </a>
              </li>
              <li
                class="btn_navbar not_visible_mobile_lg"
              >
                <div
                  class="dropdown"
                >
                  <a
                    aria-expanded="false"
                    aria-haspopup="true"
                    class="dropdown-toggle"
                    data-toggle="dropdown"
                    role="button"
                  >
                    <span>
                      pt
                    </span>
                     
                    <span
                      class="caret"
                    />
                  </a>
                  <ul
                    class="dropdown-menu"
                    id="language_selector"
                  >
                    <li>
                      <a
                        data-turbolinks="false"
                        href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                      >
                        Português
                      </a>
                    </li>
                    <li>
                      <a
                        data-turbolinks="false"
                        href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                      >
                        English
                      </a>
                    </li>
                  </ul>
                </div>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg"
              >
                <a
                  data-turbolinks="false"
                  href="/pt/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                >
                  Português
                </a>
              </li>
              <li
                class="btn_navbar only_visible_mobile_lg"
              >
                <a
                  data-turbolinks="false"
                  href="/en/items/1960-porto-portugal-alugar-babolat-pure-drive-raquete-tenis?change_locale=true"
                >
                  English
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
      <div
        id="flash_messages-anchor"
      />
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
